pkg/config/env/redis.go

```diff
@@ -1,15 +1,39 @@
 package env
 
-import "github.com/gobuffalo/envy"
+import (
+	"strconv"
+
+	"github.com/gobuffalo/envy"
+)
 
 // RedisQueuePortWithDefault returns Redis queue port used by workers defined by ATHENS_REDIS_QUEUE_PORT.
 // Standard port is 6379
 func RedisQueuePortWithDefault(value string) string {
 	return envy.Get("ATHENS_REDIS_QUEUE_PORT", value)
 }
 
-// OlympusRedisQueuePortWithDefault returns Redis queue port used by workers defined by ATHENS_REDIS_QUEUE_PORT.
+// RedisMockInMem determines whether an in-memory worker is used to mock Redis for Proxy (Athens)
+func RedisMockInMem() bool {
+	boolStr := envy.Get("ATHENS_REDIS_MOCK_IN_MEM", "false")
```

Comment:
@kdvy can you set up an environment variable that lets the user specify what kind of background workers to use? For example, what do you think of `ATHENS_BACKGROUND_WORKER_TYPE`, which could be `memory` or `redis`?

Same with the below one for olympus